src/components/reservation/helper.ts

```diff
@@ -9,4 +10,37 @@ export const getPreferredBranch = (id: string, array: AgencyBranch[]) => {
   return locationItem ? locationItem.title : id;
 };
 
+export const hardcodedInterestPeriods = (t: UseTextFunction) => {
+  return {
+    "30": t("oneMonthText"),
+    "60": t("twoMonthsText"),
+    "90": t("threeMonthsText"),
+    "180": t("sixMonthsText"),
+    "360": t("oneYearText")
+  };
+};
+
+export const getNoInterestAfter = (
+  days: number | string,
```

Comment:
Please look into how we can limit the types we access for our internal functions. A number of days should only be in the form of a number - not a string which may contain a number or not.

The fact that JavaScript object cannot have numeric keys is an implementation detail.